File: Clases2023-24/04-Repaso_Practica13_Retrofit/RetrofitFINAL_github/app/src/main/java/com/example/retrofitfinal/hotel/RatePlan.java
package com.example.retrofitfinal.hotel;

import com.google.gson.annotations.SerializedName;

public class RatePlan {
    @SerializedName("price")
    private Price price;

    public RatePlan() {
        //Empty Constructor
    }

    public Price getPrice() {
        return price;
    }

    public void setPrice(Price price) {
        this.price = price;
    }
}

File: KOTLIN_MVVM_APP/app/src/main/java/com/tokioschol/travellingkotlin/data/response/RatePlan.kt
package com.tokioschol.travellingkotlin.data.response

import com.google.gson.annotations.SerializedName

data class RatePlan(

	@field:SerializedName("features")
	val features: Features? = null,

	@field:SerializedName("price")
	val price: Price? = null
)